airframe-sql/src/main/scala/wvlet/airframe/sql/parser/SQLGenerator.scala

```diff
@@ -364,7 +364,7 @@ object SQLGenerator extends LogSupport {
       case SingleColumn(ex, alias, _, _) =>
         val col = printExpression(ex)
         alias
-          .map(x => s"${col} AS ${printExpression(x)}")
+          .map(x => s"${col} AS ${x}")
```

Comment:
If `x` contains special characters or reserved words, it should be double-quoted. 

https://github.com/wvlet/airframe/blob/d3b1fbcd386001b008c710b41dd59a2bbb5cbbf9/airframe-sql/src/main/antlr4/wvlet/airframe/sql/parser/SqlBase.g4#L438

Reply:
Never mind. x (.sqlExpr) is already double quoted